comms/src/peer_manager/node_identity.rs

```diff
@@ -221,3 +221,14 @@ impl fmt::Display for NodeIdentity {
         Ok(())
     }
 }
+
+impl fmt::Debug for NodeIdentity {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("NodeIdentity")
```

Comment:
nit: Debug usually shows all fields, so I think include all fields here but redact the ones that should never be logged:
e.g. see `base_layer/core/src/transactions/transaction_components/unblinded_output.rs`

`.field("spending_key", &"<secret>")`

It's find for the identity signature to be logged

Reply:
Add
`.field("identity_signature", &self.identity_signature)`